client.lua

```diff
@@ -126,33 +126,35 @@ function PolyZone:TransformPoint(point)
 end
 
 function PolyZone:draw()
-  local zDrawDist = 45.0
-  local oColor = self.debugColors.outline or defaultColorOutline
-  local oR, oG, oB = oColor[1], oColor[2], oColor[3]
-  local wColor = self.debugColors.walls or defaultColorWalls
-  local wR, wG, wB = wColor[1], wColor[2], wColor[3]
-  local plyPed = PlayerPedId()
-  local plyPos = GetEntityCoords(plyPed)
-  local minZ = self.minZ or plyPos.z - zDrawDist
-  local maxZ = self.maxZ or plyPos.z + zDrawDist
-
-  local points = self.points
-  for i=1, #points do
-    local point = self:TransformPoint(points[i])
-    DrawLine(point.x, point.y, minZ, point.x, point.y, maxZ, oR, oG, oB, 164)
-
-    if i < #points then
-      local p2 = self:TransformPoint(points[i+1])
-      DrawLine(point.x, point.y, maxZ, p2.x, p2.y, maxZ, oR, oG, oB, 184)
-      _drawWall(point, p2, minZ, maxZ, wR, wG, wB, 48)
+  if self.debugPoly then
```

Comment:
Can you check `self.debugGrid` here as well? If it's true we assume `debugPoly` as well (https://github.com/mkafrin/PolyZone/blob/master/client.lua#L428), so this if statement should be
```lua
if self.debugPoly or self.debugGrid then
```
to match.

Reply:
You're right. Let me know if you notice anything else.